packages/components/order-tracking/src/order-tracking.component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './order-tracking.controller';
+import template from './order-tracking.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    orderId: '<',
+    billingUrl: '<',
+    onBackButtonClick: '&',
```

Comment:
I'm not too fond of the naming `onBackButton` because you are aware that it is a `back-button` whereas it could be a modal with a cancel button for instance

Reply:
I think the problem is not the naming here but the fact that the back button is inside the component. At first i was thinking of not including the back button in the template and let the host app handle it but then i realized that the text of the back button was in the specs (back to Order N°...) so i decided to include it in the component.